module/VuFindTheme/tests/unit-tests/src/VuFindTest/ThemeInfoTest.php

```diff
@@ -244,29 +244,9 @@ public function testGetMergedConfigChild()
         $ti = $this->getThemeInfo();
         $ti->setTheme('child');
         $childJS = $ti->getMergedConfig('js');
-        $this->assertEquals(['hello.js', 'extra.js'], $childJS);
+        $this->assertEquals(['extra.js', 'hello.js'], $childJS);
```

Comment:
It could easily be either way ([just need to edit this line](https://github.com/vufind-org/vufind/pull/2543/files#diff-487ea02c96725131d0a8acbd87fd1c59bb49d0793f966115fa95541d0450ca6dR258)). Child first made more sense to me but loading order makes sense too. Which do you prefer?

Reply:
I think loading order will be less disruptive, since that's how the current behavior works.